site2/docs/reference-configuration-standalone.md

```diff
@@ -0,0 +1,1465 @@
+# Standalone
+
+## Optional
+
+### authenticateOriginalAuthData
+
+If this flag is set to `true`, the broker authenticates the original Auth data; else it just accepts the originalPrincipal and authorizes it (if required).
+
+**Default**: false
+
+### metadataStoreUrl
+
+The quorum connection string for local metadata store
+
+**Default**:
```

Comment:
Why no value? If it does not have a default value, can you add an explanation? Something like `it does not have a default value`?

Reply:
Docs for standalone is written manually, and I'm not sure about the reason why there's no default value when it should have.